packages/main/cypress/specs/Card.cy.tsx

```diff
@@ -47,14 +40,13 @@ describe("Card general interaction", () => {
 
 		// assert
 		cy.get("#cardHeader")
-			.should("have.prop", "interactive", true);
+			.click();
```

Comment:
The recommendation is to use real* methods, in this case realClick, as the methods with "real" in front are like the browser native events, not simulated like the others.